.github/workflows/auto-cc.yml

```diff
@@ -0,0 +1,32 @@
+name: "Auto Notify"
+
+on:
+  issues:
+    types: [labeled]
+  pull_request:
+    types: [labeled]
+
+jobs:
+  auto-notify:
+    if: |
+        github.repository_owner == 'cupy' &&
+        github.event.label.name == 'hip'
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Find comment
+        uses: peter-evans/find-comment@v1
+        id: fc
+        with:
+            issue-number: ${{github.event.number}}
+            body-includes: "<!-- Added by CuPy Auto Notify Bot -->"
```

Comment:
Can't the bot create a comment directly? Looks like it first creates an empty comment (this step) and then modify it (the next step). Just curious 🙂

Reply:
This step tries to find a comment including `"<!-- Added by CuPy Auto Notify Bot -->"`, and if that comment is found, the comment will be updated in the next step. This is to avoid leaving multiple comments when doing label -> unlabel -> label.

Reply:
I see, good to learn about the trick!